Cataclysm/Warlock.lua

```diff
@@ -800,7 +815,7 @@ end )
 local aliasesSet = {}
 
 spec:RegisterStateExpr( "soul_shards", function()
```

Comment:
This state expression should be removed and Soul Shards should be modeled through RegisterResource.  You can probably borrow the code from Retail.